README.md

```diff
@@ -24,29 +24,33 @@ Get a specific tree:
 
 ```graphql
 {
-  tree(id: "0x00000001"){
+  tree(id: "0x00000001") {
     id
-    hats{
+    hats {
       id
     }
   }
 }
 ```
 
 ## Hat
+
 The ID of a the hat entity is the ID of the hat in hex format.
 
 Addotionally, the hat entity includes a "prettyId" field, which is formatted in an IP address style:
+
 - In hex, with the `0x` prefix
 - Periods between the hat levels
 - Only levels with non-zero values are shown
-For example, the "prettyId" of a level 3 hat might look like this: `0x00000001.01.01`
+  For example, the "prettyId" of a level 3 hat might look like this: `0x00000001.01.01`
 
 The following query will get all the basic information of a hat:
 
 ```graphql
 {
-  hat(id: "0x0000000101000000000000000000000000000000000000000000000000000000") {
+  hat(
+    id: "0x0000000101000000000000000000000000000000000000000000000000000000"
```

Comment:
I think this should be `0x0000000100010000000000000000000000000000000000000000000000000000` now

Reply:
And same for subsequent example ids below